app.py

```diff
@@ -106,30 +112,25 @@ def __init__(self):
         self.public_key_bytes = self.public_key.public_bytes(
             encoding=serialization.Encoding.Raw, format=serialization.PublicFormat.Raw
         )
+        self.model_list = self.model_config.find_one({"name": "model_list"})["data"]
```

Comment:
It should be 
self.model_list = self.model_config.find_one({"name": "model_list"})
self.model_list = self.model_list.get("data", []) if self.model_list else [] 
In case self.model_config.find_one({"name": "model_list"}) return None